docs/source/schema/unions-interfaces.md

```diff
@@ -63,11 +71,11 @@ For more information, see [Using fragments with unions and interfaces](https://w
 
 ### Resolving a union field
 
-To fully resolve a union field for client queries, Apollo Server needs to specify which valid object type the field contains. To achieve this, you define a `__resolveType` function for the union type in your resolver map.
+To fully resolve a query that includes a union field, Apollo Server needs to specify _which_ of the union's types the field is returning. To achieve this, you define a `__resolveType` function for the union in your resolver map.
```

Comment:
Nit: "union field" -> "field which returns a union type"

Technically a union field is not a thing, though I know what you mean by it and it's less of a mouthful than what I'm suggesting. This one's at your discretion.